<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab {
  background:white;
  border: 0.5px solid black;
  padding: 10.2px;
} 

QTabWidget::pane {
  top: 1px;
  border: -px solid black;
  background-color:#f0f0f0 ; 
}</string>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Sách</string>
    </attribute>
    <widget class="QLineEdit" name="pro_input_name">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="pro_input_id">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mã sách:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tên sách:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tác giả:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pro_input_author">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="pro_input_year">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Năm phát hành:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pro_input_price">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>90</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Giá bìa:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pro_button_add">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Thêm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pro_button_update">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Sửa</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pro_input_search">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Nhập mã sách cần tìm/ xóa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pro_button_search">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>200</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Tìm kiếm</string>
     </property>
    </widget>
    <widget class="QTableView" name="pro_table_view">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>581</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pro_button_delete">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>200</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Xóa</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Nhập kho</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mã phiếu nhập:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="import_input_id">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ngày nhập:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="import_input_date">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Người nhập:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="import_input_user">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="import_pro_id">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Mã sách</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="import_pro_quantity">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Số lượng</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="import_pro_price">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string> Giá nhập</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Sản phẩm:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="import_button_add">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>90</y>
       <width>56</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Thêm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="import_button_update">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>140</y>
       <width>56</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Sửa</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="import_input_search">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Nhập mã phiếu cần tìm/ xóa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="import_button_search">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Tìm kiếm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="import_button_delete">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>180</y>
       <width>56</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Xóa</string>
     </property>
    </widget>
    <widget class="QTableView" name="import_table_view">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>581</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Xuất kho</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ngày xuất:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="export_button_search">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Tìm kiếm</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="export_input_search">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Nhập mã phiếu cần tìm/ xóa</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="export_input_user">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="export_button_update">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>140</y>
       <width>56</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Sửa</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="export_input_date">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="export_pro_id">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Mã sách</string>
     </property>
    </widget>
    <widget class="QPushButton" name="export_button_add">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>90</y>
       <width>56</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Thêm</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="export_pro_price">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Giá xuất</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mã phiếu xuất:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="export_button_delete">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>180</y>
       <width>56</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Xóa</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Người xuất:</string>
     </property>
    </widget>
    <widget class="QTableView" name="export_table_view">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>581</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="export_input_id">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Sản phẩm:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="export_pro_quantity">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Số lượng</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Kho</string>
    </attribute>
    <widget class="QTableView" name="storehouse_table_view">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>581</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="storehouse_button_search">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Tìm kiếm</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="storehouse_input_search">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Nhân viên</string>
    </attribute>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_input_id">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mã nhân viên:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pro_button_add_5">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Thêm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="user_button_add">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Thêm</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_input_name">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Tên nhân viên:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mật khẩu:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string> Vị trí:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_input_pw">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_input_role">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="user_button_update">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Sửa</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_input_search">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="user_button_search">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Tìm kiếm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="user_button_delete">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>150</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;</string>
     </property>
     <property name="text">
      <string>Xóa</string>
     </property>
    </widget>
    <widget class="QTableView" name="user_table_view">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>201</y>
       <width>581</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
